doc/dev/dataplane_codegen_quick_start.md

```diff
@@ -0,0 +1,198 @@
+# Getting Started - Generate SDK With Dataplane Codegen
+
+## Prerequisites
+
+- [Python 3.7](https://www.python.org/downloads/windows/) or later is required
```

Comment:
technically I think we're 3.6 and after for the code generator, but I'm ok with enforcing 3.7 since we want to start enforcing this everywhere